part-getting-started.tex

```diff
@@ -846,7 +846,15 @@ \subsection{Invoices with taxes included}
 \subsection{Invoices with explicit tax amounts}
 \label{subsec-sales-tax-explicit-amount}
 
+\section{Invoice Editing}
+\label{sec-invoicing-editing}
 
+Posted invoices only have 2 fields that can be edited:
+\begin{enumerate}
+\item Sales person
+\item Internal notes field 
+\end{enumerate}
+After editing, click 'Update Details' to save the changes.

```

Comment:
The field is called `Save Info`.

Reply:
I can not find 'Sales Person' in the AR -> Search -> Invoice Number -> Edit Sales Invoice form (after posting).

This change was made strictly from chat info. Evidently I did not check it.  Am I missing something?

<img width="931" alt="Screenshot 2023-04-02 at 11 37 37 AM" src="https://user-images.githubusercontent.com/1809796/229366531-7be8b535-caa7-4367-b169-ae3487df2792.png">

Reply:
The Salesperson dropdown is only shown if there are salespeople. Did you create a sales through Employees and checking the Sales checkmark?

Reply:
Nope, thanks, that is what I needed to finish checking.